src/components/Logo/Logo.js

```diff
@@ -0,0 +1,26 @@
+import React, { Fragment } from 'react'
+import logoImg from './logo-dark-bg-web.png'
+
+import './Logo.css'
+
+const CONSTANTS = {
+  alt: 'Disaster Response San Jose Logo',
+  title: 'Disaster Response SJ - Homepage'
+}
+
+export function Logo() {
+        const homeURL = window.location.origin.toString()
```

Comment:
can you explain this please?

Reply:
It's to get the homepage url so users can return to the homepage by clicking on the logo. It would always link to the root domain so they can go back to the index page when they need to.

Reply:
+1